src/main/java/core/basesyntax/service/file/FileWriterHandlerImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.file;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterHandlerImpl implements FileWriterHandler {
+
+    @Override
+    public void writeToFile(String fromSting) {
+        File fileForRaport = new File("raport.csv");
+        try (FileWriter fileWriter = new FileWriter(fileForRaport)) {
+            fileWriter.write(fromSting);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file", e);
+        }
+    }
```

Comment:
the same, you've combined IO logic with business logic. consider creating IO file reader/writer classes, and those classes that will use them and do business logic